README.md

```diff
@@ -1,6 +1,11 @@
-# Multi-threaded Google Ban Checker
+# Multi-threaded Google  Ban Checker
 
 
+# This is a Google account ban checker 
```

Comment:
I suggest, these to be added under NOTES. For e.g:

**NOTE:**
- This is not a Google account, domain or advertisements ban checker